docs/user/quick-start.md

```diff
@@ -0,0 +1,80 @@
+# Quick Start
+
+This Quick Start guide will cover:
+
+- Using kyma cli to provision control plane cluster and deploy Kyma Lifecycle Manager
+- Deploy module template
+- Using kyma cli manage modules
+
+## Prerequisites
+To use Kyma Lifecycle Manager for local setup, you need to have the following prerequisites:
+
+- [k3d](https://k3d.io/)
+- [istioctl](https://istio.io/latest/docs/setup/install/istioctl/)
+- [Kyma CLI](https://kyma-project.io/docs/kyma/latest/04-operation-guides/operations/01-install-kyma-CLI)
+
+## Provisioning a Control Plane Cluster with required resources
+
+You can provision a local k3d cluster as a control plane using the following command, by default, a cluster named `k3d-kyma` will be created. 
+```
+kyma provision k3d
+```
+Since the services deployed in the control plane are managed by Istio, you need to install Istio into the local k3d cluster using the following command:
+```
+istioctl install -y
+```
+Lifecycle Manager also exposes metrics that are collected by Prometheus Operator in the control plane to provide better observability. To simplify the local setup, you only need to deploy the ServiceMonitor CRD using the following command:
+```
+kubectl apply -f hack/servicemonitors.yaml
+```
+You can also follow official [quick start](https://prometheus-operator.dev/docs/prologue/quick-start/) guide to deploy a full set of prometheus operator into cluster as an alternative solution if you want to monitor the component performance.
+
+## Deploy Kyma lifecycle manager
+We recommend deploying Kyma Lifecycle Manager with the control plane kustomize profile, and `kcp-system`, `kyma-system` namespace must be configured upfront.
+```
+kubectl create ns kcp-system
+kubectl create ns kyma-system
+kyma alpha deploy -c alpha -k https://github.com/kyma-project/lifecycle-manager/config/control-plane
```

Comment:
why do we need the alpha channel here specifically?

Reply:
to let the generated `default-kyma` use `alpha` channel (by default, it's `regular`), so that works with the existing moduletemplate.